test/UnitTest/Microsoft.AspNet.OData.Test.Shared/PublicApi/PublicApiTest.cs

```diff
@@ -31,12 +32,25 @@ public void TestPublicApi()
             string baselineString = GetBaseLineString();
 
             // Assert
-            Assert.True(String.Compare(baselineString, outputString, StringComparison.Ordinal) == 0,
-                String.Format("Base line file {1} and output file {2} do not match, please check.{0}" +
-                "To update the baseline, please run:{0}{0}" +
-                "copy /y \"{2}\" \"{1}\"", Environment.NewLine,
-                BaseLineFileFolder + BaseLineFileName,
-                outputFile));
+            if (string.Compare(baselineString, outputString, StringComparison.Ordinal) != 0)
```

Comment:
@dxrdxr Was it necessary to include this change in this PR?

Reply:
I was getting testing errors and this simply allowed me to have better diagnostics on what was actually wrong.

I am ambivalent on if it is included, but it seemed to be a generally positive improvement to the test environment.